### ISSUE TITLE
About dataset

### ISSUE THREAD

**Issue by Mashiro1323** [shanshuo/ML-MWN#1]
您好！想请问一下您在代码中使用到的数据集，存放feature的pkl文件（包括训练集和平衡后的元验证集）是action genome官方的数据集文件，还是您在实验过程中生成的文件？如果是后者，想问您是否方便发布出来？感谢！

Hello! I would like to ask about the data set you used in your code. Is the pkl file (including training set and balanced meta-validation set)  storing feature the official data set file of action genome or the file generated during your experiment? If so, is it convenient for you to release it? Thank you very much!

**Comment by beep-bebop** [comment]
> 您好！想请问一下您在代码中使用到的数据集，存放feature的pkl文件（包括训练集和平衡后的元验证集）是action genome官方的数据集文件，还是您在实验过程中生成的文件？如果是后者，想问您是否方便发布出来？感谢！
> 
> Hello! I would like to ask about the data set you used in your code. Is the pkl file (including training set and balanced meta-validation set) storing feature the official data set file of action genome or the file generated during your experiment? If so, is it convenient for you to release it? Thank you very much!

你可以尝试写邮件给作者。

You can try try write email to author.

**Comment by shanshuo** [comment]
Apologies for the delayed response—my busy schedule has only recently allowed me some time.

The `.pkl` files in question are features generated from RPN using the Faster R-CNN as provided by [yrcong/STTran](https://github.com/yrcong/STTran). Unfortunately, due to the passage of time, I no longer have the training set `.pkl` file. However, I've made the validation set pickle file available at the following location: `data/val_videos_v2_features.pkl`.
